pages/agent/v3/cli_oidc.md

```diff
@@ -7,6 +7,16 @@ See the [OpenID Connect Core documentation](https://openid.net/specs/openid-conn
 
 <%= render "agent/v3/help/oidc_request_token" %>
 
+## OIDC URLs
+
+If using a plugin, such as our [AWS assume-role-with-web-identity](https://github.com/buildkite-plugins/aws-assume-role-with-web-identity-buildkite-plugin) plugin, you'll need to provide an OpenID provider URL, the provider URL should be set to https://agent.buildkite.com.
+
+For specific endpoints for OpenID or JWKS, see below.
+
+### Specific OIDC endpoints
+* OpenID Connect Discovery URL: https://agent.buildkite.com/.well-known/openid-configuration
+* JWKS URI: https://agent.buildkite.com/.well-known/jwks
```

Comment:
Do you think this needs it's own heading? I think it could be simplified to:

```suggestion
For specific endpoints for OpenID or JWKS, use:

* **OpenID Connect Discovery URL:** https://agent.buildkite.com/.well-known/openid-configuration
* **JWKS URI:** https://agent.buildkite.com/.well-known/jwks
```